src/mctpd.c

```diff
@@ -3867,11 +3886,13 @@ int main(int argc, char **argv)
 
 	sd_bus_flush(ctx->bus);
 	sd_bus_close(ctx->bus);
+	sd_bus_unref(ctx->bus);
```

Comment:
looks like we have a `sb_bus_flush_unref_close()` here

Reply:
Nice, thanks